### PR TITLE
Some Maintenance

### PR DESCRIPTION
**Pull request by carstene1ns**
- Add some contributors to the AUTHORS file
- Remove some bitrot (GP2X, Caanoo)
- Satisfy most `gcc -Wall` warnings
- Markdownify _all_ the things! _o/

### CODE REVIEW

**Comment by Ghabry** [comment]
btw while you are at it: This comment here: https://github.com/EasyRPG/Player/blob/master/src/window_message.h#L45  I think it can be almost fully deleted

File: README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+- autoconf >= 2.69
+- automake >= 1.11.4
+- libtool
+
+To generate the "configure" script, run before following the above section:
+
+    autoreconf -i
+
+Read more detailed instructions at:
+https://wiki.easyrpg.org/development/compiling/player/autotools
```
Comment (Ghabry) [inline README.md]: needs more linebreaks

File: README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+https://wiki.easyrpg.org/development/compiling/player/cmake
+
+### Visual Studio method:
+
+Read detailed instructions at:
+https://easy-rpg.org/wiki/development/compiling/player/windows/visual-studio
```
Comment (Ghabry) [inline README.md]: dito, is on the same line (not sure if on purpose)

File: AUTHORS.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* Diego Pedraza (zegeri)
+* Dmytro Kushnariov (rohkea)
+* Francisco de la Peña (fdelapena)
+* Gabriel Kind (ghabry)
+* Glynn Clements (glynnc)
+* Khelifi Mehdi (BlisterB)
+* Mariano Javier Suligoy (MarianoGNU)
+* Patrick Müssig (Tondorian)
+* Paulo "Zhek" Vizcaino (paulo_v)
+* scurest
```
Comment (carstene1ns) [inline AUTHORS.md]: @scurest: Now is the time to tell us how you want to be addressed here... :grinning:

**Comment by carstene1ns** [comment]
Hope there are enough linebreaks now :+1:

File: AUTHORS.md (hunk 3 above)
Comment (scurest) [inline AUTHORS.md]: Ah, just "scurest" is fine. Thanks.

Comment (Ghabry) [inline AUTHORS.md]: While you are at it, my nick is written with a capital G ;)

Comment (carstene1ns) [inline AUTHORS.md]: blame @fdelapena (https://github.com/EasyRPG/Player/commit/0332ec111c618119e14b805364040373f3c8fa6b) :dancer: yeah, will change it later.

**Comment by carstene1ns** [comment]
This should be ready.

File: src/audio_decoder.cpp [hunk 4]
```diff
@@ -122,7 +122,12 @@ std::unique_ptr<AudioDecoder> AudioDecoder::Create(FILE* file, const std::string
 
 	// Try to use MIDI decoder, use fallback(s) if available
 	if (!strncmp(magic, "MThd", 4)) {
-#ifdef HAVE_WILDMIDI
+#ifndef HAVE_WILDMIDI
+	/* WildMidi is currently the only Audio_Decoder that needs the filename passed
```
Comment (Ghabry) [inline src/audio_decoder.cpp]: wrong identation

Comment (carstene1ns) [inline src/audio_decoder.cpp]: is fixed now.

**Comment by Ghabry** [comment]
@fdelapena can you merge this. The button does not work on my stupid phone -.-

**Pull request merged**